packages/transaction-messages/src/compile/message.ts

```diff
@@ -82,6 +81,7 @@ export function compileTransactionMessage(
     const lifetimeConstraint =
         (transactionMessage as Partial<TransactionMessageWithLifetime>).lifetimeConstraint ??
         EMPTY_BLOCKHASH_LIFETIME_CONSTRAINT;
```

Comment:
Now that you have `CompiledTransaction` and `CompiledTransactionMessageWithLifetime` separated, isn't this optional now, meaning you can get rid of `EMPTY_BLOCKHASH_LIFETIME_WITH_CONSTRAINT`?